storage/storage.go

```diff
@@ -63,3 +94,26 @@ func StoreResult(result string, consumerURL string) error {
 
 	return nil
 }
+
+// createReport returns a pointer to a types.Report built from the supplied
+// map of container IDs to types.ContainerReport.
+func createReport(results map[string]types.ContainerReport) (types.Report, error) {
+	logging.Stdout("[Storage] Marshalling JSON")
+	marshalledResult, err := json.Marshal(results)
+	if err != nil {
+		return types.Report{}, err
```

Comment:
Learning question on my part: Is there an added benefit to returning an empty struct on `err`, or would `nil` work.